core-services/prow/02_config/_plugins.yaml

```diff
@@ -256,6 +256,10 @@ approve:
   repos:
   - openshift/assisted-service
   require_self_approval: false
+- commandHelpLink: ""
```

Comment:
there is no need adding this field since the value is empty.

Reply:
Added the regular value.